src/DynamoCoreWpf/Views/Search/LibraryView.xaml

```diff
@@ -89,7 +87,7 @@
                 <StackPanel Orientation="Vertical"
                             Width="72"
                             Height="72">
-                    <Border Padding="0">
+                    <Border Padding="2, 7, 2, 7">
```

Comment:
Though this work (and that I like the formatting), it is not the default and/or conventional.